civihr_employee_portal/civihr_employee_portal.install

```diff
@@ -29,13 +29,13 @@ function civihr_employee_portal_update_7001() {
     $path = 'reports/leave_and_absence/';
     $link = array();
     $link['link_path'] = $path;
-    $link['link_title'] = 'Report 2: Leave and Absence';
+    $link['link_title'] = 'Leave Reports';
     $link['menu_name'] = 'hr-reports';
     $link['router_path'] = $path . '%';
     $link['module'] = 'menu';
     $link['options'] = array(
       'attributes' => array(
-        'title' => 'CiviHR Report containing Employee, Contract, Role and Absence Activity data.',
+        'title' => 'Create reports showing the analysis of Leave taken for Holiday/Vacation, Sickness, TOIL and other leave types.',
```

Comment:
These changes were not necessary and will not have any effect. The `civihr_employee_portal_update_7001()` is an upgrader and will only be called on an existing site if it is not in version 7001 yet. All the sites are a few versions beyond this one.

As a rule of thumb, we should not update upgraders, unless there's a good reason for this (the only reason I can think of is if the upgrader has been merged to staging but has not been released to clients yet)